Cargo.toml

```diff
@@ -246,12 +246,11 @@ rusqlite = { git = "https://github.com/spiceai/rusqlite.git", rev = "97054b6af72
 # Tracking Issue: https://github.com/allan2/dotenvy/issues/113
 dotenvy = { git = "https://github.com/spiceai/dotenvy.git", rev = "e5cef1871b08003198949dfe2da988633eaad78f" }
 
-
-iceberg = { git = "https://github.com/spiceai/iceberg-rust.git", rev = "15e2fc97f27580b753c10c21e3ab5fa02d0bc4cc" }              # spiceai-0.7.0.rc1
-iceberg-catalog-glue = { git = "https://github.com/spiceai/iceberg-rust.git", rev = "15e2fc97f27580b753c10c21e3ab5fa02d0bc4cc" } # spiceai-0.7.0.rc1
-iceberg-catalog-rest = { git = "https://github.com/spiceai/iceberg-rust.git", rev = "15e2fc97f27580b753c10c21e3ab5fa02d0bc4cc" } # spiceai-0.7.0.rc1
-iceberg-datafusion = { git = "https://github.com/spiceai/iceberg-rust.git", rev = "15e2fc97f27580b753c10c21e3ab5fa02d0bc4cc" }   # spiceai-0.7.0.rc1
-iceberg_test_utils = { git = "https://github.com/spiceai/iceberg-rust.git", rev = "15e2fc97f27580b753c10c21e3ab5fa02d0bc4cc" }   # spiceai-0.7.0.rc1
+iceberg = { git = "https://github.com/spiceai/iceberg-rust.git", rev = "cdb23210224c2a3b2a8f0c82635d393df86ada6d" }              # spiceai-0.7.0.rc1
```

Comment:
This brings fix for iceberg-rust:
https://github.com/spiceai/iceberg-rust/pull/22